### DISCUSSION TITLE
Unable to install codeql pack error: (eventual cause: SunCertPathBuilderException "unable to find valid certification path to requested target")

### DISCUSSION THREAD

**Discussion by Ismailrizvi** [microsoft/Windows-Driver-Developer-Supplemental-Tools#64] [Q&A]
after cloning main branch I am trying to install codeql pack 

D:\codeql-home\codeql>codeql pack install D:\codeql-home\Windows-Driver-Developer-Supplemental-Tools\src

returns an error as "A fatal error occurred: Could not create access credentials for codeql/cpp-queries.
(eventual cause: SunCertPathBuilderException "unable to find valid certification path to requested target")"

can someone help resolve this issue?

**Comment by lauragonzalezzz** [comment]
Hi @Ismailrizvi thank you for reaching out and I am sorry you're hitting some errors with this.
I am unfamiliar with this particular error but it looks like an SSL issue from some cursory google searches so please ensure your certificates are valid.

If the certs are valid, please verify:
1. Were you able to clone CodeQL CLI?
2. Do you have access to https://github.com/github/codeql?
3. Have you tried cloning using HTTPS method instead?

Thanks,
Laura

**Reply by proitcon** [reply]
I have the exact same problem with CodeQL 2.15.4. 
The general solution seems to be to add a server certificate using OpenSSL but since this tool does not say anything about what server it downloads from it is a bit hard to fix.
